core/tests/protractor_desktop/creatorDashboard.js

```diff
@@ -123,4 +124,107 @@ describe('Creator dashboard functionality', function() {
     expect(creatorDashboardPage.getSubscribers()).toEqual('2');
     users.logout();
   });
+
+  it('should work fine in grid view as well as list view', function() {
```

Comment:
There is a sort by also right... Should we test that too?

Reply:
Yes, we should test that too. 

Should we test that in a different PR? Both are somewhat different and this PR will get long enough. I'll write the tests there in another one if that sounds good to you!

Reply:
As most of the setup would be similar I thought it would be better to just extend this test to do that. I don't mind either, but the e2e tests need not necessarily have to be very granular.

Reply:
Yes, agree they don't have to very granular. Just that it would be neat writing that functionality to different test. 

I'm also fine with both though!

Reply:
(Will open other PR for the same)

Reply:
Hmm, can we break up the test, one for grid view and one for list view?

Reply:
@kevinlee12 -- just to check, any reason why? Does the individual test run for too long as-is?

(I'm on the fence about it, so was wondering if there was a principle guiding your suggestion, or if it was ad hoc. If there is a lot of setup/teardown then we typically try to consolidate e2e tests a bit more.)

Reply:
My thinking is that this tests two separate things: grid view and list view. If we were to add tests to other views or variation of the views in the future, I don't want this block to be bloated. Also, it would be clearer to see what the failure point is if the tests were to be separated.

Reply:
I've added different tests now. Marking it as done!